Examples/DemoPlacementViewController.swift

```diff
@@ -103,25 +103,21 @@ class DemoPlacementViewController: UIViewController {
-
-        for latlon in latlons {
-            let location: CLLocation = CLLocation(latitude: latlon.0, longitude: latlon.1)
-            let sphere = SCNNode(geometry: SCNBox(width: 20.0, height: 20.0, length: 20.0, chamferRadius: 0.0))
-            sphere.geometry?.firstMaterial?.diffuse.contents = UIColor.blue
-            sphere.position = terrainNode.convertPosition(terrainNode.positionForLocation(location), to: nil)
-            sceneView?.scene?.rootNode.addChildNode(sphere)
+        
+        var locations = [CLLocation]()
+        let stride = 20 //the number of datapoints to skip between drawn locations, this cleans up the final line.
+        for index in 0..<latlons.count where index%stride == 0
+        {
+            let latlon = latlons[index]
+            locations.append(CLLocation(latitude: latlon.0, longitude: latlon.1))
         }
-
-        let firstLocation: CLLocation = CLLocation(latitude: latlons.first!.0, longitude: latlons.first!.1)
-        let startSphere = SCNNode(geometry: SCNSphere(radius: 60.0))
-        startSphere.geometry?.firstMaterial?.diffuse.contents = UIColor.green
-        startSphere.position = terrainNode.convertPosition(terrainNode.positionForLocation(firstLocation), to: nil)
-        sceneView?.scene?.rootNode.addChildNode(startSphere)
-
-        let lastLocation: CLLocation = CLLocation(latitude: latlons.last!.0, longitude: latlons.last!.1)
-        let endSphere = SCNNode(geometry: SCNSphere(radius: 60.0))
-        endSphere.geometry?.firstMaterial?.diffuse.contents = UIColor.red
-        endSphere.position = terrainNode.convertPosition(terrainNode.positionForLocation(lastLocation), to: nil)
-        sceneView?.scene?.rootNode.addChildNode(endSphere)
+        
+        let lineA = terrainNode.addPolyline(coordinates: locations, radius: 20, color: .red)
```

Comment:
Can we turn some of these values (radius, startRadius, position.y...) into variables to make it easier to change them? Not high priority, just something to think about

Reply:
I agree that these values should be editable after the line is created. As things are set up now, any change to the splines would need to trigger a new round of mesh generation in order to be visible.

It's also possible (and would be awesome) to use SCNMorpher to animate line transitions, because the we can supersample the BezierSplines to make different line geometries match up.

I think it's worth some more thought to keep things useable and efficient.